shared/ui-components/static/locales/general.json

```diff
@@ -44,7 +44,7 @@
   "listings": {
     "additionalInformation": "Additional Information",
     "allUnits": "All Units",
-    "allUnitsReservedFor": "All units reserved for {{type}}",
+    "allUnitsReservedFor": "All units reserved for %{type}",
```

Comment:
Did you mean to make all of these changes in string interpolation syntax? I don't think I have a strong opinion about what we use, but the %{str} format seems to be pretty confined to ruby.

Reply:
@bk3c So that's actually the syntax for the Polyglot library…those interpolated sections never got updated from the old Angular stuff. Figured I'd do it while in there messing with page titles.